### PR TITLE
Small corrections in the document

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

File: docs/userguide.md [hunk 1]
```diff
@@ -62,8 +62,7 @@ is a detailed list of binaries required.
 
 The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
 `kube-apiserver` >= v1.13 being installed in the path. The `kubebuilder`
-([v1.0.8](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.8)
-as of this writing) release packages all of these dependencies together.
+([v1.0.8](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.8)) release packages all of these dependencies together.
```
Comment (marun) [inline docs/userguide.md]: Why is this change suggested?

Comment (sohankunkerkar) [reply docs/userguide.md]: @marun I was under the impression that they have released v1.0.9 rc for `Kubebuilder` but it looks like that release was meant for the `controller-tools` project (sub-project for `Kubebuilder`). Let me revert this change.

File: docs/development.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ federation controller manager:
 
 ### Cleanup
 
-Follow the [cleanup instructions in the user guide](userguide.md#cleanup).
+Follow the [cleanup instructions in the user guide](README.md#uninstalling-the-chart).
```
Comment (xunpan) [inline docs/development.md]: `README.md` is not under `docs`. So, you'd better use relative links to `README.md`

Comment (sohankunkerkar) [reply docs/development.md]: done!

**Comment by marun** [comment]
/approve
LGTM

**Comment by shashidharatd** [comment]
/lgtm
Thanks @sohankunkerkar

**Pull request merged**